packages/office-ui-fabric-react/src/components/FocusZone/FocusZone.tsx

```diff
@@ -925,6 +936,126 @@ export class FocusZone extends React.Component<IFocusZoneProps> implements IFocu
     return false;
   }
 
+  private _getHorizontalDistanceFromCenter = (isForward: boolean, activeRect: ClientRect, targetRect: ClientRect): number => {
+    const leftAlignment = this._focusAlignment.x;
+    let distance = -1;
+    // ClientRect values can be floats that differ by very small fractions of a decimal.
+    // If the difference between top and bottom are within a pixel then we should treat
+    // them as equivalent by using Math.floor. For instance 5.2222 and 5.222221 should be equivalent,
+    // but without Math.Floor they will be handled incorrectly.
+    const targetRectTop = Math.floor(targetRect.top);
+    const activeRectBottom = Math.floor(activeRect.bottom);
+    const targetRectBottom = Math.floor(targetRect.bottom);
+    const activeRectTop = Math.floor(activeRect.top);
+    // for paging down
+    if (isForward && targetRectTop < activeRectBottom) {
```

Comment:
This check and the `else if` check could be pulled into one variable and then you could check that in the `if` and remove the `else if`... And it looks like at that point the `if` could return and then the current body of the `else` doesn't need to be in an else anymore